pom.xml

```diff
@@ -175,9 +192,14 @@
                                 <artifactId>maven-checkstyle-plugin</artifactId>
                         </plugin>
                         <plugin>
-                                <groupId>org.codehaus.mojo</groupId>
-                                <artifactId>findbugs-maven-plugin</artifactId>
-                                <version>3.0.4</version>
+                                <groupId>com.github.spotbugs</groupId>
+                                <artifactId>spotbugs-maven-plugin</artifactId>
+                                <version>${spotbugs.version}</version>
+                        </plugin>
+                        <plugin>
+                                <groupId>org.apache.maven.plugins</groupId>
+                                <artifactId>maven-project-info-reports-plugin</artifactId>
+                                <version>${maven.project.info.reports.plugin.version}</version>
                         </plugin>
```

Comment:
you need jxr plugin as well to see the source linked to the reports

eg.
`<plugins>
...
 <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-jxr-plugin</artifactId>
                </plugin>
...
</plugins>`